### ISSUE TITLE
[Quest] Subject to Interpretation - ID 11991 Grizzly Hills 

### ISSUE THREAD

**Issue by Drostthehunter** [WoWManiaUK/Blackwing-Lair#1356]
**Links:**
https://www.wowhead.com/quest=11991/subject-to-interpretation
https://www.wowhead.com/item=35799/frozen-mojo (!)
https://www.wowhead.com/item=35797/drakurus-elixir
**What is happening:**
I have to kill Drakkari Warriors and Drakkari Shamans to get Frozen Mojo (!) in order to use 
https://www.wowhead.com/item=35797/drakurus-elixir to summon Image of Drakuru, but i can not kill the mobs bcs i cant attack them  ( Mouseovering over the targeted NPCs does not show the "sword" icon and non the AoE spells can aggro them.

**What should happen:**
I should be able to kill those mobs normally. 

**What i have noticed**
This problem of not being able to target those specific NPCs occured while i was doing the other quest related to this group of mobs. The quest was: https://www.wowhead.com/quest=11984/filling-the-cages. 
Long story short: Once i took Budd from the camp to help me cage one of those NPCs they werent targetable but i was able to complete the quest just by putting the https://www.wowhead.com/item=35736/bounty-hunters-cage underneath them.
